src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -475,13 +475,14 @@ private static string GetDocumentFilePathFromName(string documentName)
             };
         }
 
-        private static LSP.DidOpenTextDocumentParams CreateDidOpenTextDocumentParams(Uri uri, string source)
+        private static LSP.DidOpenTextDocumentParams CreateDidOpenTextDocumentParams(Uri uri, string source, string? languageId)
```

Comment:
If we're just going to coerce this to an empty string, then maybe just
```suggestion
        private static LSP.DidOpenTextDocumentParams CreateDidOpenTextDocumentParams(Uri uri, string source, string languageId = "")
```